### ISSUE TITLE
MoveIt Calibration Noetic Release (needs a version number)

### ISSUE THREAD

**Issue by JStech** [ros-planning/moveit_calibration#32]
[_moveit.ros.org release documentation_](https://moveit.ros.org/documentation/contributing/releases/)

### Checklist
- [ ] Create `noetic-devel` branch
- [ ] Update changelogs
- [ ] Run ROS buildfarm prerelease test
- [ ] Bloom
- [ ] Write discourse post

**Comment by JStech** [comment]
Since [crigroup/handeye](https://github.com/crigroup/handeye) is not yet released for Noetic, it looks like this will have to wait.

**Comment by felixvd** [comment]
Are they on it? I think Francisco (the maintainer) is not with crigroup anymore.

**Comment by JStech** [comment]
I've heard nothing so far. Personally I'd prefer to use a C++ implementation of the solver, since it'd make the code way cleaner, but @RoboticsYY [reported that the Python implementation gave a better final result](https://github.com/ros-planning/moveit/pull/1559#discussion_r392312061).

Typically I wouldn't be in a rush to get a noetic release out, but the annoyances with OpenCV 3.2 in melodic make it a bit more pressing, I feel.

**Comment by felixvd** [comment]
Now that the samples can be extracted, you could test a few implementations and see if that impression was well-founded.

**Comment by JStech** [comment]
Well, @RoboticsYY shared the results of that investigation with me, and the CRI group solvers do generally outperform the alternatives that were available at the time. (Want to see for yourself? Watch our [ROS World](https://roscon.ros.org/world/2020/) talk!).

OpenCV was not included in that investigation, so I'm still inclined to implement a solver plugin that uses it. In the meantime, though, the handeye package is being (or has been?) released to Noetic and ROS2, so we can move ahead with a Noetic release. Also, I think I now prefer to leave the existing sovler plugin as-is, and put the OpenCV solvers in a new plugin, giving users more options.

**Comment by JStech** [comment]
I think we can revive this, as the CRI Group packages are in the pipeline for a Noetic release ([whenever that happens](https://discourse.ros.org/t/can-we-get-a-noetic-sync-please/17240/4)). I just built everything in Noetic, with just those three CRI Group packages built from source, so I think it's good to go.

**Comment by JStech** [comment]
The CRI Group packages have been released (after a bit of a hiccup). I'm gonna try to get this release moving again.

**Comment by JStech** [comment]
@tylerjw IIRC this stalled because I can't push to the release repo or something, and then we figured we don't need yet another PickNikker doing MoveIt release stuff. Think you could take over getting this across the finish line?

**Comment by tylerjw** [comment]
Yes, I'll create an internal issue to track this along with the other releases I need to do.

**Comment by JStech** [comment]
@tylerjw ping
